foundry/pixi/containers/ruler.d.ts

```diff
@@ -1,167 +1,184 @@
-declare interface RulerData {
-  _state: number;
-  class: string;
-  destination: PIXI.Point;
-  name: string;
-  waypoints: [PIXI.Point];
-}
-
 /**
  * The Ruler - used to measure distances and trigger movements
- * @param user - The User for whom to construct the Ruler instance
+ * @param user  - The User for whom to construct the Ruler instance
```

Comment:
I think we put parameters to the constructor now most of the time (at least that's what I did). Not 100% sure though if that's the right thing to do.